Integrations/EWSMailSender/EWSMailSender.py

```diff
@@ -315,9 +307,21 @@ def main():
         else:
             return_error(error_message + '\n' + debug_log)
     finally:
-        if isinstance(config, Configuration):
-            # The protocol will not kill its threads after use, so kill it manually
-            config.protocol.thread_pool.terminate()
+        try:
+            # we don't want to leave cached connection around as EWS limits the number of connections
+            # in a very aggressive way. 12 seems to be the default limit
+            # see: https://blogs.msdn.microsoft.com/webdav_101/2018/06/02/you-are-doing-too-much-at-one-time-ewsmaxconcurrency-too-many-concurrent-connections-opened/ # noqa
+            close_connections()
+        except Exception as ex:
+            demisto.info("Failed close_connections (shouldn't happen). Ignoring exception: {}".format(ex))
```

Comment:
should this be `demisto.error` ?

Reply:
We had it before on info. We can change.